js/classes/zone.js

```diff
@@ -26,6 +26,11 @@ class Zone {
         this.load = () => {
             // Clear all entities from previous Zone.
             GAME.clearEntities();
+
+            // Add entities essential to any Zone.
```

Comment:
That's a good call.

Reply:
It's possible there is a better place to do this, @B1GD3VD0G would know better than me. It made sense to me to do it here though.